Jenkinsfile

```diff
@@ -60,7 +60,7 @@ pipeline {
       steps {
         script {
           if (env.BRANCH_NAME == 'master') {
-            deploy("/web.integreat-app.de")
+            deploy("/integreat.app")
```

Comment:
Not sure, if this will work. @maxammann ?

Reply:
Yep!